components/nominations/NomineeCard.tsx

```diff
@@ -1,9 +1,15 @@
+import { Clickable } from "components/types";
+import { Nominee } from "data/types";
 import Image from "next/image";
 import React from "react";
 
 import styles from "styles/modules/NomineeCard.module.scss";
 
-const NomineeCard = ({ data, onClick }) => {
+interface NomineeCardProps extends Clickable {
+  data: Nominee;
+}
+
+const NomineeCard = ({ data, onClick }: NomineeCardProps) => {
```

Comment:
I wonder if we could do something like
```suggestion
// Put interface in types.ts
interface NomineeData {
  data: Nominee;
}

const NomineeCard = ({ data, onClick }: NomineeData & Clickable) => {
```
Then, NomineeModalProps (in NomineeModal.tsx) would become:
```ts
interface NomineeModalProps extends NomineeData {
  roles: string[];
  isOpen: boolean;
  toggle: () => void;
}
```

Reply:
I always prefer the interface notation rather than intersections because it is much easier to add new params to.

Reply:
Feedback has been implemented in the form of an empty interface